src/shared/lib/api/youtube-endpoints.ts

```
@@ -4,12 +4,17 @@ export const YOUTUBE_CHANNEL_PLAYLIST_VIDEOS = (
 ) =>
   `https://www.googleapis.com/youtube/v3/playlistItems?part=snippet,contentDetails,status&playlistId=${playlistId}&maxResults=${limit}&key=${process.env.YOUTUBE_API_KEY}`;
 
-export const YOUTUBE_CHANNELS_INFORMATION = (
+export const YOUTUBE_CHANNEL_INFORMATION_BY_IDS = (
   channelIds: string[],
   limit: number = 25
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)